packages/block-library/src/media-text/index.php

```diff
@@ -48,6 +48,7 @@ function render_block_core_media_text( $attributes, $content ) {
 	}
 	$processor->set_attribute( 'src', esc_url( $current_featured_image ) );
 	$processor->set_attribute( 'class', 'wp-image-' . get_post_thumbnail_id() . ' size-' . $media_size_slug );
+	$processor->set_attribute( 'alt', esc_attr( get_post_meta( get_post_thumbnail_id(), '_wp_attachment_image_alt', true ) ) );
```

Comment:
```suggestion
	$processor->set_attribute( 'alt', trim( strip_tags( get_post_meta( get_post_thumbnail_id(), '_wp_attachment_image_alt', true ) ) ) );
```

This might be better. Because it matches [the internal implementation](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/media.php#L1076) of the `wp_get_attachment_image()` function for getting the HTML of the image element. Additionally, it is consistent with [the implementation of the Post Featured Image block](https://github.com/WordPress/gutenberg/blob/ee3fd1a6756b3cd2cd254092c2e879ceb2cc5ec9/packages/block-library/src/post-featured-image/index.php#L31).